includes/ocos.h

```diff
@@ -119,12 +108,12 @@ extern FxLoadCustomOpFactory LoadCustomOpClasses_Math;
 
 #ifdef ENABLE_TOKENIZER
 extern FxLoadCustomOpFactory LoadCustomOpClasses_Tokenizer;
-#endif // ENABLE_TOKENIZER
+#endif  // ENABLE_TOKENIZER
 
 #ifdef ENABLE_TF_STRING
 extern FxLoadCustomOpFactory LoadCustomOpClasses_Text;
 #endif  // ENABLE_TF_STRING
 
-#ifdef ENABLE_OPENCV
-extern FxLoadCustomOpFactory LoadCustomOpClasses_OpenCV;
+#ifdef ENABLE_CV2
```

Comment:
Why don't we follow this if/def patterns to manage all enable_op flags in one unified header?

Defining an extern variable here is to avoid including each operator file in ort_customops.cc.

Reply:
Are we using the extern because we're concerned about the impact on build time by including a header for the factory that includes the declaration of the individual operators? Or are there other negative consequences of the operator declarations begin included in ort_customops.cc?

Maybe @edgchen1 can comment on best practices for usage of `extern` in C++ code. Historically I've avoided it unless necessary as `extern` statements can get out of sync, but maybe I'm overlooking some benefits of it.

Reply:
Because we don't have a set test to verify enable/disable CMake options,  I avoid including all header files in one file(ort-customops.cc) to detected unintentionally macro/function reference among these operator implementations, build speed is a bonus.

The prototype sync is a protentional issue in some cases, which in this header, we can only use a name, and out-of-sync will cause link error, so it should be good.

Reply:
will note that the Google C++ style guideline prefers including headers to forward declarations:
https://google.github.io/styleguide/cppguide.html#Forward_Declarations

we (aim to) follow these guidelines in onnxruntime, not sure what the coding standards for onnxruntime-extensions are